packages/central-server/src/database/createPermissionsBasedMeditrakSyncQueue.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { SqlQuery } from '@tupaia/database';
+
+/**
+ * Creates the permissions_based_meditrak_sync_queue Materialized View
+ *
+ * This is used to improve the speed of querying the meditrak_sync_queue when
+ * doing a permissions based sync
+ */
+const query = new SqlQuery(`
```

Comment:
Big ol' SQL query yay! Lemme know if you'd prefer me to abstract this into something more high level? I don't mind it as it stands cos it's more readable than it would be otherwise.